po/template.pot

```diff
@@ -196,10 +266,22 @@ msgstr ""
 msgid "Expires"
 msgstr ""
 
+#: src/js/controllers/importLegacy.js
+msgid "Failed to import wallets"
+msgstr ""
+
 #: public/views/create.html
 msgid "Family vacation funds"
 msgstr ""
 
+#: src/js/controllers/walletHome.js
+msgid "Fetching Payment Informantion"
```

Comment:
When you make a payment using BIP72 (Payment Protocol URI), Copay try to get information from the URL. So, this message will show to the user while Copay downloads information about the payment (address, amount, merchant, etc...).